tests/common/utils.rs

```diff
@@ -146,7 +146,7 @@ pub fn hash_outpoints(sending_data: &HashSet<Outpoint>) -> Scalar {
 }
 
 pub fn verify_and_calculate_signatures(
-    privkeys: Vec<SecretKey>,
+    privkeys: Vec<Scalar>,
```

Comment:
nit: let's replace the `privkeys` by `tweaks` here

Reply:
done